### PR TITLE
Use codacy for code coverage

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -13,36 +13,16 @@
...
-  - ./mvnw -Pfabric8 install
-  post:
-  - mkdir -p $CIRCLE_TEST_REPORTS/junit/
-  - find . -type f -regex ".*/target/.*-reports/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
-  - cp runtime/target/classes/static/index.html $CIRCLE_ARTIFACTS/apidocs.html
-  - cp runtime/target/classes/static/swagger.json $CIRCLE_ARTIFACTS/
-  - cp runtime/target/classes/static/swagger.yaml $CIRCLE_ARTIFACTS/
-  - bash <(curl -s https://codecov.io/bash)
-
-deployment:
```
Comment (rhuss) [inline circle.yml]: The deployment step is not needed anymore ?

File: code-reports/pom.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rhuss) [inline code-reports/pom.xml]: Couldn't this be also done directly in the top-level pom with a profile ?

Or what other purpose does this module serve ?

File: circle.yml (hunk 1 above)
Comment (jimmidyson) [inline circle.yml]: This was an experiment...

File: code-reports/pom.xml (hunk 2 above)
Comment (jimmidyson) [inline code-reports/pom.xml]: No jacoco can only aggregate modules built before it in the reactor so can't be in the parent.

**Pull request closed without merging**